packages/react-devtools-shared/src/utils.js

```diff
@@ -317,6 +318,25 @@ export function setBreakOnConsoleErrors(value: boolean): void {
   );
 }
 
+export function getHideDoubleLogsInStrictMode(): boolean {
```

Comment:
This is a super tiny naming nit: "double logs" and "in strict mode" is a little redundant. Maybe `getHideConsoleLogsInStrictMode` (and `LOCAL_STORAGE_HIDE_CONSOLE_LOGS_IN_STRICT_MODE` and `hideConsoleLogsInStrictMode`)?